libs/vm_monitor/src/cgroup.rs

```diff
@@ -654,8 +654,10 @@ impl CgroupWatcher {
             .set_mem(cgroups_rs::memory::SetMemory {
                 min: None,
                 low: None,
-                high: Some(MaxValue::Value(limits.high.min(i64::MAX as u64) as i64)),
-                max: Some(MaxValue::Value(limits.max.min(i64::MAX as u64) as i64)),
+                high: Some(MaxValue::Value(
+                    u64::min(limits.high, i64::MAX as u64) as i64
+                )),
+                max: Some(MaxValue::Value(u64::min(limits.max, i64::MAX as u64) as i64)),
```

Comment:
Just realizing these _might_ be clearer with `Ord::clamp` (e.g. `limits.high.clamp(0, i64::MAX as u64)`). Not sure though. Also ok as it is

Also could make sense to factor out `i64::MAX as u64` into a local variable because it's being used a bunch here.

These don't need to be done in this PR though :)

Reply:
Yeah I was thinking about clamp as well. It's a little bit extra, but it's the most clear in my mind. Tbh I think about min/max in terms of clamp.